friends/src/App.js

```diff
@@ -0,0 +1,41 @@
+import React, { Component } from 'react';
+import './App.css';
+import axios from 'axios';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state ={
+    friends: [],
+    error: ''
+    };
+  }
+
+  componentDidMount() {
+    axios
+    .get("http://localhost:5000/friends")
+    .then(res => {
+      console.log(res);
+      this.setState({ friends: res.data });
+    })
+    .catch(err=> {
+      this.setState({ error: err });
```

Comment:
Nice use of error handling, it would be good to create a conditional render in your JSX that actually displays the error to the screen to the user what went wrong with your get request to the server.